src/sample_submission_data.R

```diff
@@ -0,0 +1,9 @@
+## Script that generates sample submission data to pass validation checks
+require(arrow) # for parquet writing
```

Comment:
```suggestion
here::i_am("src/sample_submission_data.R")
require(arrow) # for parquet writing
```

Reply:
It is indeed appropriate here (pun intended): 

Information about the package can be found at https://here.r-lib.org

Reply:
You really want the script to fail here if you don't have arrow. `library()` throws an error if the package is not found while `require()` merely returns `FALSE`.  

```suggestion
library("arrow") # for parquet writing
```